security-proxy/README.md

```diff
@@ -3,20 +3,77 @@
 The security-proxy (aka SP) belongs to geOrchestra core, since it is the component which :
  * handles user sessions
  * routes requests to webapps (except CAS)
- 
+
 The behavior is controlled by the files from the `<datadir_root>/security-proxy` folder, which can be found [here](https://github.com/georchestra/datadir/tree/master/security-proxy)
 
+## How-to integrate a new application in geOrchestra ?
+
+The goal here is to take advantage of the [SSO](https://en.wikipedia.org/wiki/Single_sign-on) feature for the new application without having to "CASify" it.
+
+### Proxy configuration
+
+It may sound obvious, but the new application has to be proxified by the security-proxy first !
+This can be configured in the [targets-mapping](https://github.com/georchestra/datadir/blob/master/security-proxy/targets-mapping.properties) file, which can be found in geOrchestra datadir. Remember: changes to this file require the security proxy to be restarted.
```

Comment:
It can be done in the [targets-mapping](https://github.com/georchestra/datadir/blob/master/security-proxy/targets-mapping.properties) file, which can be found in geOrchestra datadir. Remember: changes in  this file requires to restart  the security proxy.